Doc/ncsdk 33212 (#23036)
nrfconnect/sdk-nrf: doc/nrf/app_dev/device_guides/nrf54l/fota_update.rst

         For samples using random HCI identities, the Test and Confirm mode should not be used.

   #. Wait for the DFU to finish and then verify that the new application works properly by observing the new device name visible in the Device Manager app.

>    #. After confirming if the advanced tool mode was used, reset the device again. This ensures that the new application runs correctly and allows you to revoke old signature keys if desired.

Reviewer: ```suggestion
   #. After confirming if the advanced tool mode was used, reset the device again. 
      This ensures that the new application runs correctly and allows you to revoke old signature keys if desired.
```

         For samples using random HCI identities, the Test and Confirm mode should not be used.

   #. Wait for the DFU to finish and then verify that the new application works properly by observing the new device name visible in the Device Manager app.

>    #. It is recommended to reset the device once again (after confirmation if the tool advance mode was used) to ensure that the new application is running correctly.
>       This step also allows to revoke old signature keys if desired.

Reviewer: please double-check the meaning

```suggestion
   #. After confirming if the advanced tool mode was used, reset the device again.
   This ensures that the new application runs correctly and allows you to revoke old signature keys if desired.
```
